src/Audio/Cue.cs

```diff
@@ -774,6 +783,14 @@ internal bool INTERNAL_update()
 								long elapsedFromStop = INTERNAL_timer.ElapsedMilliseconds - INTERNAL_fadeStart;
 								variableValue = elapsedFromStop;
 							}
+							else
+							{
+								variableValue = 0.0f;
```

Comment:
Since variableValue is unconditionally computed on line 768, this ensures that it is held at position 0 until it's time.